service/history/workflow/task_generator.go

```diff
@@ -218,6 +236,34 @@ func (r *TaskGeneratorImpl) GenerateWorkflowCloseTasks(
 	return nil
 }
 
+func (r *TaskGeneratorImpl) getArchivalTaskDelay(namespaceEntry *namespace.Namespace, retention time.Duration) time.Duration {
```

Comment:
Please check and reuse `backoff.JitDuration`.

Reply:
Fixed, thanks for pointing that out.